Drv/CMakeLists.txt

```diff
@@ -14,4 +14,8 @@ add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriver/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/LinuxSerialDriver/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/LinuxSpiDriver/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/LinuxI2cDriver/")
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Ip/")
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TcpClient/")
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TcpServer/")
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Udp/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/SocketIpDriver/")
```

Comment:
`SocketIpDriver` should not be needed in the Drv/cMakeLists.txt after upgrading to new ipv4 drivers.